requirements_docs.txt

```diff
@@ -10,3 +10,9 @@ xmlrpc2==0.3.1
 
 git+https://github.com/pypa/pypa-docs-theme.git#egg=pypa-docs-theme
 git+https://github.com/python/python-docs-theme.git#egg=python-docs-theme
+
+# This is needed since autodoc imports all bandersnatch packages and modules
+# so imports must not fail or its containing module will NOT be documented.
+# Also, the missing swift dependencies will cause the doc build to fail since
+# autodoc will raise a warning due to the import failure.
+-r requirements_swift.txt
```

Comment:
This should be *fine* as long people are OK with installing 40+ dependencies (I didn't count, but it's a LOT)